docs/installation.md

```diff
@@ -0,0 +1,125 @@
+## Installation
```

Comment:
I generally don't like duplication because it's hard to keep them the same. Another option would be to keep the readme the same but change the installation guide into a customization guide, which only contains the additional pieces of information that most people won't need.